integr8ly/installation
# INTLY-986: Put back che uninstall (with minor improvements).

## Additional Information
JIRA: https://issues.jboss.org/browse/INTLY-986

## Verification Steps
Run the uninstall playbook.
Verify that codeready namespace has been removed.
Run the uninstall playbook again and check exist code(should be 0).
